src/Collection.php

```diff
@@ -478,6 +480,23 @@ public function getNamespace()
     }
 
     /**
+     *
+     * @see Grop::__construct()
+     * @param mixed $keys Fields to group by. If an array or non-code object is passed, it will be the key used to group results.
+     * @param array $initial Initializes the aggregation result document.
+     * @param Javascript $reduce An aggregation function that operates on the documents during the grouping operation.
+     * @param array $options Command options
+     * @return Traversable
+     */
+    public function group($keys, array $initial, Javascript $reduce, array $options = [])
```

Comment:
It is make sense especially if `group` could be deprecated. But if this library implements an API similar to that of the legacy PHP driver, may be it would be better to add this helper? Any way i removed this commit from current PR and created a separate branch for it.